src/main/webapp/creatives/maps/UserActivityLogger.js

```diff
@@ -16,7 +16,7 @@ class UserActivityLogger {
    * @param {string} url A URL to send the request to.
    */
   sendRequest_(url) {
-    fetch(`/${url}?correlator="${this.correlator_}"`);
+    fetch(`${URL.BASE_URL}/${url}?correlator=${this.correlator_}`);
```

Comment:
It reads a bit weird to have `URL.BASE_URL/url`, maybe rename the url param to `path` or similar